packages/stylelint-config/package.json

```diff
@@ -1,15 +1,32 @@
 {
-  "name": "@lmc-eu/stylelint-config",
+  "name": "@almacareer/stylelint-config",
   "version": "8.0.0",
-  "description": "LMC's config for Stylelint",
+  "description": "Alma Career's config for Stylelint",
   "keywords": [
     "stylelint",
-    "lmc",
-    "config"
+    "config",
+    "alma",
+    "career",
+    "preset"
   ],
   "author": "Tomáš Litera <tomas.litera@lmc.eu>",
-  "license": "BSD-3-Clause",
+  "license": "MIT",
+  "type": "module",
   "main": "index.js",
+  "files": [
+    "index.js",
+    "rules/extras.js",
+    "rules/order.js",
+    "rules/scss.js"
+  ],
+  "exports": {
+    ".": {
+      "import": {
+        "default": "./index.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
```

Comment:
What kind of magic is this? 🙂 Is it related to the `eslint-import-resolver-node` dependency in the repository root?

Reply:
No, this is a different (modern) approach to define the package entry points. See: https://nodejs.org/api/packages.html#exports